### PR TITLE
Feature/event feedbacks

### PR DESCRIPTION
**Pull request by kr2006**

### CODE REVIEW

File: server/services/seeders/20200320142421-eventFeedback.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            feedback: 'I am waiting for the next time. It was so funny. Thanks!',
+            date: 1584916013000
+          },
+          {
+            user_event_id: 7,
+            feedback: 'Good, but not enough..',
+            date: 1584516013000
+          }
+        ], {});
+
```
Comment (yushevchuk) [inline server/services/seeders/20200320142421-eventFeedback.js]: Remove space, please.

File: client/src/events/components/Feedbacks/LeaveFeedbacks.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            value: '',
+            isValid: false
+        }
+    })
+    
+    const submitFeedbackHandler = (event) => {
+        event.preventDefault();
+        addFeedback();
+    }
+
```
[the remark this answers is not in the document]
Comment (kr2006) [reply client/src/events/components/Feedbacks/LeaveFeedbacks.js]: Done

File: client/src/events/components/Feedbacks/LeaveFeedbacks.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    {headers}
+                ) 
+                history.replace(location.pathname, {rerender:true});
+            }   
+            catch (err) {
+                console.log(err);
+            }
+        }
+    }
+
```
[the remark this answers is not in the document]
Comment (kr2006) [reply client/src/events/components/Feedbacks/LeaveFeedbacks.js]: Done

File: client/src/events/components/Feedbacks/ReadFeedbacks.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+    useEffect(()=>{
+        getFeedbacks()
+    },[])
+
+    useEffect(()=>{
+        getFeedbacks();
+        history.replace(location.pathname, {rerender: false})
+    },[props.flag])
+
```
[the remark this answers is not in the document]
Comment (kr2006) [reply client/src/events/components/Feedbacks/ReadFeedbacks.js]: Done

File: client/src/shared/components/UI/ScrollToElement.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
```
[the remark this answers is not in the document]
Comment (kr2006) [reply client/src/shared/components/UI/ScrollToElement.js]: Done

File: server/controllers/eventController.js [hunk 6]
```diff
@@ -1,19 +1,18 @@
...
 const UserEvent = require('../models').user_event;
-const Category = require('../models').category;
+const Feedbacks = require('../models').event_feedback;
 const Redis = require('../services/redisService');
 
 const STATUS_ACTIVE = 'Active';
 const STATUS_BANNED = 'Banned';
 const STATUS_DELETED = 'Deleted';
+const CURRENT_DATE = new Date().getTime();
+
```
[the remark this answers is not in the document]
Comment (kr2006) [reply server/controllers/eventController.js]: Done

File: server/services/seeders/20200320142421-eventFeedback.js (hunk 1 above)
Comment (kr2006) [reply server/services/seeders/20200320142421-eventFeedback.js]: Done

**Pull request merged**